src/bindings/python/__init__.py

```diff
@@ -51,22 +51,27 @@ class Object:
         setattr(base if new_base is None else new_base, enum, obj)
 
 
-convert(Annotator, 'Type', [
+convert(enums, 'CellMLElement', [
         'COMPONENT',
         'COMPONENT_REF',
         'CONNECTION',
         'ENCAPSULATION',
         'IMPORT',
-        'ISSUE',
         'MAP_VARIABLES',
+        'MATHML',
         'MODEL',
         'RESET',
         'RESET_VALUE',
         'TEST_VALUE',
+        'UNDEFINED',
         'UNIT',
         'UNITS',
         'VARIABLE'
-        ])
+], libcellml)
+
+# The Annotator class relies on the CellMLElmement declaration above.
+from libcellml.annotator import Annotator
+
```

Comment:
I don't understand why this is here?

Reply:
Because of the transformation of how SWIG does enums and how we want to make them more friendly in Python.

Reply:
... but does it need to be in this file?  Now that the enum for Type has been moved out of the Annotator class?